README.markdown

```diff
@@ -6,6 +6,42 @@ standards such as [JSON Schema](http://json-schema.org), [JSON
 Pointer](https://www.rfc-editor.org/rfc/rfc6901),
 [JSONL](https://jsonlines.org), and more.
 
+Example
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()  {
+    // Creating JSON using parse
+    sourcemeta::jsontoolkit::JSON document =
+        sourcemeta::jsontoolkit::parse(R"JSON({
+            "name": "John Doe",
+            "age": 20,
+            "address": "zxy"
```

Comment:
Maybe put a short semi-real address? I guess you can generate a supposedly valid i.e. US address on Google